notebooks/data-sources/TestGrid/metrics/unexpected_test_failures.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Can you print some examples of jobs that have a high unexpected failure rate according to this method? And some that are expected failures?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>